bin/setenv-docker-customize.sh

```diff
@@ -154,6 +155,11 @@ EXO_ES_URL="${EXO_ES_SCHEME}://${EXO_ES_HOST}:${EXO_ES_PORT}"
 
 [ -z "${EXO_REGISTRATION}" ] && EXO_REGISTRATION="true"
 
+[ -z "${EXO_CLUSTER}" ] && EXO_CLUSTER="false"
+[ -z "${EXO_CLUSTER_NAME}" ] && EXO_CLUSTER_NAME="${HOSTNAME}"
+[ -z "${EXO_CLUSTER_HOSTS}" ] && EXO_CLUSTER_HOSTS="-"
+[ -z "${EXO_JGROUPS_ADDR}" ] && EXO_JGROUPS_ADDR="GLOBAL"
```

Comment:
Yes, it's a jgroups binding option :
(from http://www.jgroups.org/manual4/index.html#Transport)
> The bind address which should be used by this transport. The following special values are also recognized: GLOBAL, SITE_LOCAL, LINK_LOCAL, NON_LOOPBACK, match-interface, match-host, match-address